packit/cli/packit_base.py

```diff
@@ -55,11 +55,17 @@ def get_command(self, ctx, cmd_name):
         "If this is not specified, default to the first remote."
     ),
 )
+@click.option(
+    "-c",
+    "--config",
+    "package_config_path",
+    help="Path to package configuration file (defaults to .packit.yaml or packit.yaml)",
```

Comment:
```suggestion
    help="Path to package configuration file (aka packit.yaml)",
```

B/c there are actually many more names this could have than just `.packit.yaml` or `packit.yaml`.